apps/betterangels-backend/accounts/types.py

```diff
@@ -130,12 +145,14 @@ class UserBaseType:
 
 @strawberry_django.type(User)
 class UserType(UserBaseType):
+    # TODO: has_accepted_tos, has_accepted_privacy_policy, is_outreach_authorized shouldn't be optional.
+    # Temporary fix while we figure out type generation
     id: ID
-    username: auto
+    has_accepted_tos: Optional[bool]
```

Comment:
`has_accepted_tos`, `has_accepted_privacy_policy`, `is_outreach_authorized` shouldn't be optional. This is a temporary fix while we figure out type generation. Without `Optional`, these fields must be included on every user query (including queries wherein `user` is a field, e.g., client profile, note) to avoid type errors. I assume this means we're doing something wrong because, in a graphql query, all available fields are optional; no field _must_ be included on a request